backend/src/server/routes/v3/secret-router.ts

```diff
@@ -166,6 +166,11 @@ export const registerSecretRouter = async (server: FastifyZodProvider) => {
         workspaceSlug: z.string().trim().optional().describe(RAW_SECRETS.LIST.workspaceSlug),
         environment: z.string().trim().optional().describe(RAW_SECRETS.LIST.environment),
         secretPath: z.string().trim().default("/").transform(removeTrailingSlash).describe(RAW_SECRETS.LIST.secretPath),
+        expand: z
```

Comment:
I think it might be better if we call this `expandSecretReferences`. This way anyone looking at it won't have to be like expand what? What do you think?

Reply:
resolved it